src/PKSim.Core/Mappers/PopulationPKAnalysisToDataTableMapper.cs

```diff
@@ -37,7 +37,7 @@ private void addAnalysisToTable(PopulationPKAnalysis analysis, DataTable dataTab
 
       private string getCurveName(PopulationPKAnalysis analysis)
       {
-         return analysis.CurveData.Caption;
+         return analysis.CurveData.Caption + analysis.ExtraDescription;
```

Comment:
use string interpolation instead of concatenating strings. Not a big deal here but just something to simply do all the time so that strings are not allocated for nothing